delta/sdk-testkit/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/SseEventLogDummy.scala

```diff
@@ -0,0 +1,50 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.testkit
+
+import akka.persistence.query.Offset
+import cats.syntax.functor._
+import ch.epfl.bluebrain.nexus.delta.kernel.Mapper
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Event.{OrganizationScopedEvent, ProjectScopedEvent}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.organizations.OrganizationRejection
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.{ProjectRef, ProjectRejection}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{Envelope, Event, Label}
+import ch.epfl.bluebrain.nexus.delta.sdk.{JsonLdValue, SseEventLog}
+import fs2.Stream
+import monix.bio.{IO, Task, UIO}
+// $COVERAGE-OFF$
+
+final class SseEventLogDummy(envelopes: Seq[Envelope[Event]], f: PartialFunction[Event, JsonLdValue])
```

Comment:
As we don't really care about the returned values which should be tested elsewhere (and if we remove the need of an envelope as I mentioned in another comment), we could just have:
```
final class SseEventLogDummy(events: (Label, Org, JsonLdValue))
```
And filter the list in the project / org method

Reply:
We need the envelopes, as I pointed out

Reply:
I'm open to something like that If it makes things more readable. So let's consider that for a separate PR if we arrive to a better solution.